Azure/azure-cli
# [RDBMS] Add List-SKUS Command, Table Transformers, Local Context for Postgres, MySQL, Mariadb Single Server

**Description**  

1. Added list-skus commands for single server Postgres and MySQL
2. Added table transformers for list-skus and server list commands
3. Added local-context support for single server


**Testing Guide**  

1. List-skus command: run with the below commands
- az postgres server list-skus -l eastus -o table
- az mysql server list-skus -l eastus -o table
- az mariadb server list-skus -l eastus -o table

2. server list command: run with the below commands
- az postgres server list -g <group_name> -n <server_name> -o table
- az mysql server list -g <group_name> -n <server_name> -o table
- az mariadb server list -g <group_name> -n <server_name> -o table

3. local context command: 
az config param-persist on
- az postgres -h and run commands to check if server name argument is included in the local context
e.g.) az postgres server create -g <group_name> -n <server_name> -l <location> => az postgres server show

**History Notes**  


---

This checklist is used to make sure that common guidelines for a pull request are followed.
